README.rst

```diff
@@ -34,21 +45,92 @@ cloud and run the analysis to obtain (5) output results.
 -------------
 
 In this example, the input datasets representing the collision and simulated
-data will be generated. Therefore there is no explicit input data to be taken
-care of.
+data will be generated on the fly in the first coupld of workflow steps.
+Therefore there is no explicit input data to be taken care of.
 
 2. Analysis code
 ----------------
 
 This example uses the `ROOT <https://root.cern.ch/>`_ analysis framework with
-the custom user code located in the ``code`` directory:
-
-- `generantuple.py <code/generantuple.py>`_ - generate N-tuples
-- `hepdata_export.py <code/hepdata_export.py>`_ - prepare HepData submission
-- `histogram.py <code/histogram.py>`_ - generate histogram
-- `makews.py <code/makews.py>`_ - create channel samples
-- `plot.py <code/plot.py>`_ - create final plots
-- `select.py <code/select.py>`_ - apply shape variations
+the custom user code located in the ``code`` directory. In order to
+execute the different stages of the analysis a number of scripts are needed.
+In a real analysis these scripts might be part of larger analysis frameworks
+developed using the experiment-internal software stack (e.g. CMSSW or the ATLAS
+Analysis Releases) and be based on C++ with many dependencies and require multiple
+container images. In this emulation we will stick to python and ROOT only that
+can all be run in the same environment.
```

Comment:
Actually we have two environments:

```console
$ git grep image workflow/ | sort -u
workflow/steps.yml:    image: reanahub/reana-demo-bsm-search
workflow/steps.yml:    image: reanahub/reana-env-root6
```

Maybe tune down to say something like "In this emulation example we shall use two different Python and ROOT environments."